less_users/models.py

```diff
@@ -41,19 +38,7 @@ def total_points(self):
                 total += log.points
         return total
 
-    @property
-    def days_left(self):
-        """return the rest of the substraction of challenge duration and timedelta between activation date and today;
-        ex. duration: 1 month (30 days), activation date: 01.01.2020, today: 28.01.2020.
-        gives: 30 - (28 - 1) = 3 """
-        start = int(self.start_date.date().strftime("%Y%m%d"))
-        today = self.todays_day_num()
-        challenge_duration = self.challenge.duration
-        days_left = challenge_duration - (today - start)
-        return int(days_left)
-
-    @property
-    def get_points(self):
+    def get_points(self, date_today=date.today()):
```

Comment:
❗

Reply:
dodałam to tylko, żeby móc wrzucać argument inny niż faktycznie dzisiejsza data do testów, ale widzę, że przez to mi się sypie cały kod 😞  nie wiem co z tym zrobić.

Reply:
Co rozumiesz przez "sypie mi się cały kod"?

Reply:
Wspominasz o dwóch problemach, które nie dotyczą mojego komentarza:
- używanie `@property` nie ma nic tutaj do wywoływania funkcji.
- możesz wywołać freeze_time nie jako dekorator a jako context manager, wtedy możesz mieć różne daty w różnych częściach testu. takie zastosowanie jest pokazane w dokumentacji to tej biblioteki.


> A sypie się, bo nawet Ty zwracasz mi uwagę, że gdy ustawiam wartość zmiennej w argumentach na "date.today()", to nie działa jak należy...

[Dwa tygodnie temu](https://github.com/justinaluc/LESS_WASTE/pull/3#discussion_r1074028600) dostałaś w tej samej linijce komentarz do którego się nie odniosłaś. 
Dodatkowo, "sypie się" nie jest opisem co konkretnie nie działa, dlatego ciężko mi w jakikolwiek sposób nakierwować cię na rozwiązanie.

Reply:
Widzę, że już jesteś zniecierpliwiona moimi tłumaczeniami. Ale ja naprawdę nie wiem jak wytłumaczyć niektóre zagadnienia pisząc, a nie pokazując o co chodzi i co mi nie działa. 
Usiłuję naprawić działanie z datami, tzn. nie chcę wprowadzać daty przy sprawdzaniu check_if_active, ani get_points. 
Już udało mi się w testach wykorzystać freezera, który przeskakuje w czasie i tak sprawdzać działanie tych funkcji.
Teraz mi z kolei nie działa Log. Ale do jutra mam nadzieję, że znajdę błąd.

Reply:
Jeżeli reviewer mówi Ci, że jakąś zmianą wprowadzasz buga do kodu, to musisz to naprawić. Jeżeli nie rozumiesz o co dokładnie chodzi to warto zapytać o rozjaśnienie problemu.
Nie jestem zniecierpliwiona Twoimi tłumaczeniami, chciałabym po prostu, żebyś w wątkach adresowała problemy, których te wątki dotyczą, a nie wspominała o innych.

Natomiast od rozmów od innych problemów jest slack - tam jak najbardziej możesz mnie zapytać, pokazać kawałek kodu który Ci nie działa, powiedzieć czego od niego oczekujesz, a co dostajesz. Fajnie też jakbyś podawała jakie sposoby rozwiązywania problemu już zastosowałaś.
Naprawdę ciężko jest coś wydedukować z "sypie się".

Reply:
OK. Postaram się tak działać.